protocol/message.go

```diff
@@ -19,13 +19,49 @@ type QuotedMessage struct {
 	Text string `json:"text"`
 }
 
+type CommandState int
+
+const (
+	CommandStateRequestAddressForTransaction CommandState = iota + 1
+	CommandStateRequestAddressForTransactionDeclined
+	CommandStateRequestAddressForTransactionAccepted
+	CommandStateRequestTransaction
+	CommandStateRequestTransactionDeclined
+	CommandStateTransactionPending
+	CommandStateTransactionSent
+)
+
+type CommandParameters struct {
+	// ID is the ID of the initial message
+	ID string `json:"id"`
+	// From is the address we are sending the command from
+	From string `json:"from"`
+	// Address is the address sent with the command
+	Address string `json:"address"`
+	// Contract is the contract address for ERC20 tokens
+	Contract string `json:"contract"`
+	// Value is the value as a string sent
+	Value string `json:"value"`
+	// TransactionHash is the hash of the transaction
+	TransactionHash string `json:"transactionHash"`
+	// CommandState is the state of the command
+	CommandState CommandState
```

Comment:
We don't want to have `json:"commandState"` here? Or omit it?

Reply:
Forgot to change it, I was testing with the capital letter, good point